src/pip/_internal/index/package_finder.py

```diff
@@ -700,6 +700,11 @@ def trusted_hosts(self):
         for host_port in self._link_collector.session.pip_trusted_origins:
             yield build_netloc(*host_port)
 
+    @property
+    def cert(self):
+        # type: () -> Optional[str]
+        return getattr(self._link_collector.session, "verify", None)
```

Comment:
hmmmm, test failures suggest that I'm grabbing a bool here instead of the cert path... I was going off of this:
https://github.com/pypa/pip/blob/master/src/pip/_internal/cli/req_command.py#L103-L105

but seems I'm missing some intermediate processing somewhere...

Reply:
ah I see, this is probably just a `requests` param, which defaults to true